src/ArcanaPrime.sol

```diff
@@ -30,6 +31,8 @@ import {OperatorFilterer} from "closedsea/src/OperatorFilterer.sol";
 error MaxQuantityAllowedExceeded();
 error MaxEntitlementsExceeded();
 error MaxSupplyExceeded();
+error MintSupplyExceeded();
+error RemainingSupplyExceeded();
```

Comment:
Custom error is not used anywhere, should be removed

Reply:
done